openvpn/froot/ca-east.frootvpn.ovpn

```diff
@@ -1,18 +1,22 @@
 client
 dev tun
-proto udp
 resolv-retry infinite
-remote-random
 nobind
 persist-key
-persist-tun
-auth-user-pass /config/openvpn-credentials.txt
-ns-cert-type server
+auth-user-pass
```

Comment:
This line should not replace `auth-user-pass /config/openvpn-credentials.txt`
The username and password supplied as enviroment variables are persisted to file and needs to be read so that OpenVPN don't prompt for credentials on startup.